### PR TITLE
Wallet popup without transaction

### PR DESCRIPTION
**Pull request by ShawTim**
image

### CODE REVIEW

File: packages/tempus-client_v3/src/i18n/translations/it.json [hunk 1]
```diff
@@ -174,6 +174,10 @@
   "WalletErrorModal": {
     "errorModalDescription": "Un errore è occorso con il tuo wallet, per favore controllalo e riprova."
   },
+  "WalletPopup": {
+    "title": "",
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/it.json:178]: `Panoramica del Wallet`

File: packages/tempus-client_v3/src/i18n/translations/it.json [hunk 2]
```diff
@@ -174,6 +174,10 @@
...
+    "titleAddress": ""
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/it.json:179]: `Wallet connesso`

**Comment by iamvukasin** [comment]
Reviewing

File: packages/tempus-client_v3/src/i18n/translations/zz.json [hunk 3]
```diff
@@ -174,6 +174,10 @@
   "WalletErrorModal": {
     "errorModalDescription": "~~~~~====={{{{{(((((errorModalDescription)))))}}}}}=====~~~~~"
   },
+  "WalletPopup": {
+    "title": "",
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/i18n/translations/zz.json:178]: Translation

File: packages/tempus-client_v3/src/components/Wallet/WalletPopup.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  .tc__tooltip {
+    z-index: 9999;
+    background-color: var(--walletPopupBackground);
+    width: 328px;
+    padding: 16px;
+    transform: translate(-100%, calc(100% + 8px));
+    box-sizing: border-box;
+    border: none;
+    border-radius: 5px;
+    box-shadow: 0px 4px 8px var(--walletPopupShadow);
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/components/Wallet/WalletPopup.scss:30]: It's effectively same:
```suggestion
    box-shadow: var(--downStrongShadow);
```

**Review by iamvukasin — APPROVED** [review]

File: packages/tempus-client_v3/src/i18n/translations/es.json [hunk 5]
```diff
@@ -174,6 +174,10 @@
   "WalletErrorModal": {
     "errorModalDescription": "Ocurrió un error con su wallet, verifíquelo e intente nuevamente."
   },
+  "WalletPopup": {
+    "title": "",
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/es.json:178]: `Visión general del wallet`

**Review by danieledamiani — DISMISSED** [review]

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**